src/libraries/Common/src/Interop/Unix/System.Native/Interop.GetCpuUtilization.cs

```diff
@@ -10,12 +10,12 @@ internal static partial class Sys
         [StructLayout(LayoutKind.Sequential)]
         internal struct ProcessCpuInformation
         {
-            internal ulong lastRecordedCurrentTime;
-            internal ulong lastRecordedKernelTime;
-            internal ulong lastRecordedUserTime;
+            internal double lastRecordedCurrentTime;
```

Comment:
These fields should stay as `ulong`

Reply:
Okay, would you please explain why?

Reply:
The time is measured in ticks. Integer is the common natural representation for it. For example, `System.TimeSpan` or `System.DateTime` use integer to represent time.

Also, you have kept the time as integer in the Windows-specific implementation: https://github.com/dotnet/runtime/pull/66743/files#diff-3b49513abffadf32b251d645b79126456245b1b5134ae44f3895b24496eef684R8-R11 , so it makes sense to keep it as integer in the Unix specific implementation too.